src/features/unified-share-modal/SharedLinkAccessDescription.js

```diff
@@ -15,26 +17,53 @@ type Props = {
 };
 
 const SharedLinkAccessDescription = ({ accessLevel, enterpriseName, itemType }: Props) => {
+    const getDescriptionForAnyoneWithLink = (type: ItemType) => {
+        switch (type) {
+            case ITEM_TYPE_HUB:
+                return messages.peopleWithLinkSignedInRequiredDescription;
+            default:
+                return messages.peopleWithLinkDescription;
```

Comment:
Instead of adding an additional prop for signed in required, we are only showing signed-in required for `item type=hubs` since Hubs doesn't support anonymous users for shared links. 

We will remove this switch statement once Hubs supports anonymous users.